proxy/proxy/client.go

```diff
@@ -317,7 +317,7 @@ func (c *Client) startRefresh(instance string, refreshCfgBuffer time.Duration) c
 			timeToRefresh = certExpiration.Sub(now) - (5 * time.Second)
 			logging.Errorf("new ephemeral certificate expires sooner than expected (adjusting refresh time to compensate): current time: %v, certificate expires: %v", now, certExpiration)
 		}
-		logging.Infof("Scheduling refresh of ephemeral certificate in %s", timeToRefresh)
+		logging.Infof("Scheduling refresh of ephemeral certificate in %s", timeToRefresh.Round(time.Millisecond))
```

Comment:
Let's just round to time.Second. The millisecond portion doesn't help much.

https://go.dev/play/p/g_fNIWzOi8J

Reply:
Sure. Done.